packages/rancher-istio/charts/app-readme.md

```diff
@@ -23,9 +23,23 @@ To limit scraping to specific namespaces, set `prometheus.prometheusSpec.ignoreN
 
 **Custom Prometheus Installation with Kiali**
 
-To use a custom Monitoring installation, set the `kiali.external_services.prometheus` url in the values.yaml. This url depends on the values for `nameOverride`, `namespaceOverride`, and `prometheus.service.port`:
+To use a custom Monitoring installation, set the `kiali.external_services.prometheus` url in the values.yaml. This url depends on the values for `nameOverride`, `namespaceOverride`, and `prometheus.service.port` in your rancher-monitoring or other monitoring instance:
 ```
 http://{{ .Values.nameOverride }}-prometheus.{{ .Values.namespaceOverride }}.svc:{{ prometheus.service.port }}
 ```
+**Custom Grafana Installation with Kiali**
+
+To use a custom Grafana installation, set the `kiali.external_services.grafana` url in the values.yaml. This url depends on the values for `nameOverride`, `namespaceOverride`, and `granfa.service.port` in your rancher-monitoring or other grafana instance:
+```
+http://{{ .Values.nameOverride }}-grafana.{{ .Values.namespaceOverride }}.svc:{{ grafana.service.port }}
+```
+**Custom Tracing Installation with Tracing**
```

Comment:
with Kiali ?

Reply:
Its an external link to the grafana instance from kiali, similar to the prometheus one. I was seeing errors in the ui without this set.

Reply:
I mean all of these say "custom blah install with kiali", not "with tracing".